src/api/schema/sort.rs

```diff
@@ -15,16 +15,14 @@ use crate::api::schema::{
 };
 
 #[derive(Enum, Copy, Clone, PartialEq, Eq)]
+#[derive(Default)]
```

Comment:
**nit**: This PR's been inconsistent with adding the `derive(Default)` on a new line or on the existing one.

Reply:
I think `cargo clippy --fix` added it on a new line, and the manual ones I had to do added it to the existing line. I'll fix them

Reply:
That's what I expected!